cities/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import relationship
+
+from database import Base
```

Comment:
Ensure that the `Base` object is correctly imported from the `database` module. This is crucial for the model to be correctly mapped to the database.